lib/interop.js

```diff
@@ -363,9 +363,18 @@ Interop.prototype.toUnifiedPlan = function(desc) {
     // In the iteration that follows, we use the cached Unified Plan (if it
     // exists) to assign mids to ssrcs.
 
+    var type;
+    if (desc.type === 'answer') {
+        type = 'offer';
+    } else if (desc.type === 'offer') {
+        type = 'answer';
+    } else {
+        throw new Error("Type '" + desc.type + "' not supported.");
+    }
+
```

Comment:
The point is that the cached description typically has the inverse type of the description we want traduce, so in the test added in this commit cached is never set.